src/replication/replication_slave_node.cpp

```diff
@@ -265,12 +265,31 @@ namespace cubreplication
     stop_and_destroy_online_repl ();
   }
 
+  void slave_node::destroy_transfer_receiver ()
+  {
+    delete m_transfer_receiver;
+    m_transfer_receiver = NULL;
+  }
+
+  void slave_node::wait_fetch_completed ()
+  {
+    // this forces transefer_receiver to stream::commit_append all data it has received
+    destroy_transfer_receiver ();
+
+    if (m_lc != NULL)
+      {
+	m_stream->prepare_fetch_all ();
+	// Need to wait for a notification signifying that everything was fetched before calling
+	// m_stream->stop ()
```

Comment:
remove comments

Reply:
they are relevent, with m_stream->stop I wanted to say something, it is not code that was commented out. You already had this in mind or you thought it was code commented out?

Reply:
I thought it was old code.
Maybe if there was a plain text explanation instead of code-text it wouldn't have been misleading.